behaviortree_ros2/src/tree_execution_server.cpp

```diff
@@ -91,7 +91,6 @@ TreeExecutionServer::TreeExecutionServer(const rclcpp::NodeOptions& options)
 
   // register the users Plugins and BehaviorTree.xml files into the factory
   RegisterBehaviorTrees(p_->params, p_->factory, p_->node);
```

Comment:
As @kenyoshizoe pointed out we can't load the Tree.xml files that are pass in via parameters before the statically linked behaviors are loaded.